rel/overlay/etc/default.ini

```diff
@@ -47,7 +47,7 @@ changes_doc_ids_optimization_threshold = 100
 max_document_size = 8000000 ; bytes
 ;
 ; Maximum attachment size.
-; max_attachment_size = infinity
+; max_attachment_size = 1073741824 : 1 gibibyte
```

Comment:
Use `;` instead of `:` to introduce comment.


`; max_attachment_size = 1073741824 : 1 gibibyte`
............................................................... ^----- should be `;`

Reply:
Done in fa8288bdd